code/script.js

```diff
@@ -2,18 +2,146 @@
 
 // Step 1 - Welcome and introduction
 // Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+alert (
```

Comment:
There should be no space between JavaScript methods and their corresponding parentheses. Please go through your whole file and change where it applies.